src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,14 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        String qw = "1234";
```

Comment:
You should create UserService object, call findByEmail method and return true if user by email exists and passed password is equal to user's password.
Return false in any other cases.